### DISCUSSION TITLE
Best practice and workflow for approving and publishing

### DISCUSSION THREAD

**Discussion by howaskew** [AstunTechnology/os-custom-geonetwork#6] [Q&A]
What's the recommended approach to manage creating, editing, approving and publishing records in a single geonetwork instance? How to robustly distinguish works in progress from records for public consumption?

**Comment by archaeogeek** [comment]
https://www.geonetwork-opensource.org/manuals/trunk/en/user-guide/publishing/index.html might help- I'll try and get some more documentation down for you next week though :-)
